pkg/config/config.go

```diff
@@ -521,8 +521,13 @@ func InitConfig(config Config) {
 	config.BindEnvAndSetDefault("cri_query_timeout", int64(5))      // in seconds
 
 	// Containerd
-	// We only support containerd in Kubernetes. By default containerd cri uses `k8s.io` https://github.com/containerd/cri/blob/release/1.2/pkg/constants/constants.go#L22-L23
-	config.BindEnvAndSetDefault("containerd_namespace", "k8s.io")
+	if IsKubernetes() {
+		// By default containerd cri uses `k8s.io` https://github.com/containerd/cri/blob/release/1.2/pkg/constants/constants.go#L22-L23
+		config.BindEnvAndSetDefault("containerd_namespace", "k8s.io")
```

Comment:
I'm not sure we need to limit here. Watching all namespaces will also work in Kubernetes.

Reply:
You're right. It's no longer needed. I added a new commit to simplify this.